### PR TITLE
Stop theme aggregation from aggregating node_module files incorrectly.

### PR DESCRIPTION
**Pull request by yuderekyu**
Fix

## Verification
clinical pull request: https://github.com/cerner/terra-clinical/pull/560
dev-site pull request: https://github.com/cerner/terra-dev-site/pull/203

### CODE REVIEW

**Comment by emilyrohrbough** [comment]
Can we open a test PR in a few projects to validate?

**Review by bjankord — APPROVED** [review]

**Comment by yuderekyu** [comment]
updated description to contain links to terra-clinical verification pull request and terra-dev-site pull request

**Review by mjhenkes — APPROVED** [review]

File: scripts/aggregate-themes/theme-aggregator.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ class ThemeAggregator {
     const assets = ThemeAggregator.find(`**/themes/${themeName}/${file}`, options);
 
     // Add the dependency import if it exists.
-    assets.unshift(...ThemeAggregator.find(`${NODE_MODULES}${theme}/**/${file}`, options));
+    assets.unshift(...ThemeAggregator.find(`${NODE_MODULES}${themeName}/**/${file}`, options));
```
Comment (emilyrohrbough) [inline scripts/aggregate-themes/theme-aggregator.js:56]: should there be a test that verifies this fix behavior?

Comment (emilyrohrbough) [reply scripts/aggregate-themes/theme-aggregator.js:56]: Is this what caused the lib & src from being aggregated as well?

Comment (yuderekyu) [reply scripts/aggregate-themes/theme-aggregator.js:56]: Test case - https://github.com/cerner/terra-toolkit/pull/335/commits/8f3aa29281f762e4f25cb7c178fa3c1b69f1bddd No, this fixes a separate issue.

`theme` can either be the default theme or null. Without this change, we will either attempt to aggregate default node_module theme assets for every theme specified in config, or attempt to aggregate any theme file :(

**Review by emilyrohrbough — APPROVED** [review]

**Pull request merged**